tests/NClient/NClient.Tests/ClientTests/OptionalParamClientTest.cs

```diff
@@ -11,6 +11,17 @@ namespace NClient.Tests.ClientTests
     [Parallelizable]
     public class OptionalParamClientTest
     {
+        [Test, Order(0)]
+        public void OptionalParamClient_Build_NotThrow()
+        {
+            const string anyHost = "http://localhost:5000";
```

Comment:
I think we should use Fixture or smth fot generate some general values..

Reply:
Maybe not right here and now, just fyi

Reply:
I will use them. I just forgot that Fixture is already used in tests for deconstructors.

Reply:
Done